### PR TITLE
validated all resources against API2800 for Terraform 0.11

### PR DESCRIPTION
**Pull request by AsisBagga**
### Description
validated all resources against API2800 and while validating few resource cause Terraform to crash which has been fixed in this PR. 

### Issues Resolved
N/A

### Check List
- [x] New functionality includes testing.
  - [x]  All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ ] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## Build Status 
 
 OV Version | 6.00 | 5.60 | 5.50 | 5.40 | 5.30 |
```
Comment (yuvirani) [inline README.md:5]: Add column for 6.10

File: examples/certificates_server/data_source.tf [hunk 2]
```diff
@@ -3,15 +3,15 @@ provider "oneview" {
...
+  ov_apiversion = 2800
   ov_ifmatch = "*"
 }
 
 # Testing Data Source 
 # Any one of the these fields can used to get data source
 data "oneview_server_certificate" "sc" {
 #         alias_name = "hm_cert"
-         remote_ip = "<Server_IP>"
+         remote_ip = "172.18.11.11" //"<Server_IP>"
```
Comment (yuvirani) [inline examples/certificates_server/data_source.tf:14]: Remove IP's

File: examples/hypervisor_managers/main.tf [hunk 3]
```diff
@@ -3,26 +3,26 @@ provider "oneview" {
...
  description = "Hypervisor Manager IP"
- default = "<HM-IP>"
+ default = "172.18.13.11" //"<HM-IP>"
 }
 
 variable "hm_username" {
  type = "string"
  description = "Hypervisor Manager Username"
- default = "<HM-Username>"
+ default = "dcs" //"<HM-Username>"
```
Comment (yuvirani) [inline examples/hypervisor_managers/main.tf:19]: Dont expose the credentials

File: README.md (hunk 1 above)
Comment (AsisBagga) [reply README.md:5]: will take care of this in Release PR

**Review by yuvirani — APPROVED** [review]

**Review by VenkateshRavula — APPROVED** [review]

**Pull request merged**